pkg/metrics/metrics.go

```diff
@@ -0,0 +1,138 @@
+package metrics
+
+import (
+	logging "github.com/op/go-logging"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	subsystem = "asb"
+)
+
+var (
+	sandboxCreated = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: subsystem,
+			Name:      "sandbox_created",
+			Help:      "Counter of all sandbox namespaces that are created.",
```

Comment:
I can't really comment on the validity of this metrics file :) I'm going to have to trust you that this is the proper way of doing it :)